### ISSUE TITLE
YAPP_Demo programs @rosenhauer

### ISSUE THREAD

**Issue by mrWheel** [mrWheel/YAPP_Box#68]
@rosenhauer,

Is it an idea to name all programs that test/demo a specific function off the YAPPgenerator the name "YAPP_Demo_&lt;array_name&gt;.scad" and also reflect this in the comment block at the beginning of the program.

All not needed array's should be removed from the YAPP_Demo_program so it is clear what is needed to (only) create (in this case) boxMounts.

All these YAPP_Demo programs should then be part of the repo.

Something like:
```
//-----------------------------------------------------------------------
// Yet Another Parameterized Projectbox generator
//
//  This is the YAPP_Demo_boxMounts program for testing and
//  demonstrate the use of the boxMounts array.
//
//  Needs YAPPgenerator Version 3.0
//
//-----------------------------------------------------------------------
```

## Why?
To give users a simple example for using the (getting more and more complex) YAPPgenerator.

**Comment by rosenhauer** [comment]
Good idea.  The can be used also to generate the screenshots for the documentation.
